charts/camunda-platform/templates/tasklist/deployment.yaml

```diff
@@ -184,6 +187,10 @@ spec:
           configMap:
             name: {{ include "tasklist.fullname" . }}
             defaultMode: {{ .Values.tasklist.configMap.defaultMode }}
+        - name: config-tasklist
```

Comment:
Is it needed to duplicate the volume itself? As I remember we can use the same CM in different places.